hw/ip/otbn/data/otbn.ld.tpl

```diff
@@ -52,7 +52,16 @@ SECTIONS
 
         *(.data*)
         . = ALIGN(32);
+
+        /* Align section end (see note in .text section) */
+        . = ALIGN(4);
```

Comment:
I don't think this is needed: the previous line aligns to 32 bytes, right?

Reply:
This was already in the `.data` section -- the diff is just showing it here because I added an identical clause to `.bss`. Perhaps it's not needed in either?

Reply:
Yep, but the code used to be:
```
        *(.data*)
        . = ALIGN(32);
        *(.bss*)

        /* Align section end (see note in .text section) */
        . = ALIGN(4);
```
and your patch removes the `*(.bss*)` line which means there's no longer any way of "unaligning" things after the `ALIGN(32)`.

Reply:
Ah, I see -- I'll fix that, thanks!

Reply:
The original reason was to allow people to use e.g. `BN.LID` on the last stuff in the section without having to worry about getting integrity errors from uninitialised memory just above. But (a) Don't do that! and (b) We're probably zeroing all of memory in the first place anyway, so it won't be a problem.

Anyway, that's a long-winded way of saying "sounds good to me" :-)